docs/guides/cli/05_cli-basics.mdx

```diff
@@ -26,7 +25,7 @@ Available Commands:
 
 Flags:
   -c, --config string   Path to the Ory Network configuration file.
-      --format string   Set the output format. One of table, json, and json-pretty. (default "default")
+      --format string   Set the output format. One of table, json, yaml, json-pretty, jsonpath and jsonpointer. (default "default")
```

Comment:
Did you copy that from the actual output?

Reply:
Yes I literally copied the output from my command line:

```
~ > ory auth --help
Create a new Ory Network account or sign in to an existing account.

Usage:
  ory auth [flags]
  ory auth [command]

Available Commands:
  logout      Signs you out of your account on this computer.

Flags:
  -c, --config string   Path to the Ory Network configuration file.
      --format string   Set the output format. One of table, json, yaml, json-pretty, jsonpath and jsonpointer. (default "default")
  -h, --help            help for auth
  -q, --quiet           Be quiet with output printing.
  -y, --yes             Confirm all dialogs with yes.

Use "ory auth [command] --help" for more information about a command.
```

also didnt see jsonpath and jsonpointer before 😅